spec/support/fake_symphony.rb

```diff
@@ -32,6 +32,16 @@ class FakeSymphony < Sinatra::Base
     json_response 200, "patron/#{params[:key]}.json"
   end
 
+  get '/symws/rest/patron/lookupPatronInfo' do
+    content_type :'text/html'
+    status 200
+    begin
+      File.open(File.dirname(__FILE__) + '/fixtures/patron/' + "#{params['userID']}.xml").read
```

Comment:
Awesome that you figured this out.  I wonder if we should put these files under a separate directory (like `/fixtures/patron/payment_history` or something like that so that we can distinguish these files from the files that are meant to represent a web services response.